mods/ra/rules/vehicles.yaml

```diff
@@ -714,6 +714,12 @@ QTNK:
 	SelectionDecorations:
 		VisualBounds: 44,38,0,-4
 	MadTank:
+	Detonates:
+		ChargeDelay: 96
+		DetonationDelay: 42
+		ChargeSound: MADCHRG2.AUD
+		DetonationSound: MADEXPLO.AUD
```

Comment:
There's no opinion here: these aud files are named lower case on disk, so changing the case breaks things for people with case sensitive file systems.